packages/web/src/pages/dashboard-page/store/sagas.ts

```diff
@@ -67,11 +73,24 @@ function* fetchDashboardAsync(
         limit,
         getUnlisted: true
       }),
-      call(audiusBackendInstance.getPlaylists, accountUserId, [])
+      call([sdk.full.users, sdk.full.users.getPlaylistsByUser], accountUserId),
+      call([sdk.full.users, sdk.full.users.getAlbumsByUser], accountUserId)
     ])
-    // Casting necessary because yield* all is not typed well
     const tracks = data[0] as Track[]
-    const playlists = data[1] as Collection[]
+    const playlists = transformAndCleanList(
+      (data[1] as Awaited<ReturnType<typeof sdk.full.users.getPlaylistsByUser>>)
+        .data,
+      userCollectionMetadataFromSDK
```

Comment:
I am surprised this doesn't get upset about data possibly being `null | undefined`. I guess the transformer handles that case...